server/services/educationAndWorkPlanService.ts

```diff
@@ -74,9 +76,10 @@ export default class EducationAndWorkPlanService {
     return this.educationAndWorkPlanClient.unarchiveGoal(unarchiveGoalDto.prisonNumber, unarchiveGoalRequest, token)
   }
 
-  async getEducation(prisonNumber: string, token: string): Promise<EducationDto> {
+  async getEducation(prisonNumber: string, username: string): Promise<EducationDto> {
```

Comment:
This is the right change to make 👍 

But you'll also need to change your middleware `retrieveEducation` because that is the function that calls this function, so you need to change it to pass the username instead of the user token.